src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -163,6 +163,14 @@ public function note($message)
         $this->block($message, 'NOTE', 'fg=yellow', ' ! ');
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function info($message)
+    {
+        $this->block($message, 'INFO', 'fg=green', ' ', true);
```

Comment:
I have no data to back this one up, but I would associate `blue` with info and `green` with success.

Reply:
I noticed that info is light bluish in [bootstrap](https://getbootstrap.com/docs/4.0/components/alerts/) but I replicated what I got by using $io->writeln(<INFO>%S</INFO>);

Reply:
All right, thanks for the clarification.

While I still would change every "INFO" to blue-ish, it definitely makes sense to use the same style as `<INFO></>` here. 👍